includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1182,25 +1182,31 @@ public function request_capability( string $capability_id, bool $requested ) {
 	 * @param array  $site_data      - Data to track ToS agreement.
 	 * @param array  $actioned_notes - Actioned WCPay note names to be sent to the on-boarding flow.
 	 * @param array  $account_data   - Data to prefill the progressive onboarding.
+	 * @param bool   $progressive_complete - Whether we need to redirect user to Stripe KYC to complete their payouts data.
 	 *
 	 * @return array An array containing the url and state fields.
 	 *
 	 * @throws API_Exception Exception thrown on request failure.
 	 */
-	public function get_onboarding_data( $return_url, array $business_data = [], array $site_data = [], array $actioned_notes = [], $account_data = [] ) {
+	public function get_onboarding_data( $return_url, array $business_data = [], array $site_data = [], array $actioned_notes = [], $account_data = [], $progressive_complete = false ) {
 		$request_args = apply_filters(
 			'wc_payments_get_onboarding_data_args',
 			[
-				'return_url'          => $return_url,
-				'business_data'       => $business_data,
-				'site_data'           => $site_data,
-				'create_live_account' => ! $this->is_in_dev_mode(),
-				'actioned_notes'      => $actioned_notes,
-				'progressive'         => ! empty( $account_data ),
-				'account_data'        => $account_data,
+				'return_url'           => $return_url,
+				'business_data'        => $business_data,
+				'site_data'            => $site_data,
+				'create_live_account'  => ! $this->is_in_dev_mode(),
+				'actioned_notes'       => $actioned_notes,
+				'progressive'          => ! empty( $account_data ),
+				'progressive_complete' => $progressive_complete,
+				'account_data'         => $account_data,
 			]
 		);
 
+		if ( $progressive_complete ) {
+			unset( $request_args['force_on_boarding'] );
```

Comment:
I wonder if that's just for testing purposes 🤔

Reply:
oh, yeah, I'll remove it 🙈 good catch!

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/4848/commits/c8d1dd310c2042c8e0ac947cb7c8bccf95cd41e2